src/ts/Html5Http.ts

```diff
@@ -1,28 +1,20 @@
-///<reference path="../ts/Conviva.d.ts"/>
-/*! (C) 2016 Conviva, Inc. All rights reserved. Confidential and proprietary. */
```

Comment:
All TS files that contain(ed) this header are files from the Conviva SDK, that were simply converted to TS but otherwise untouched, so please keep all those headers within the files. Don't make their code seem like ours.

Reply:
@dweinber @stonko1994

Reply:
we decided together with @dweinber to remove those comments from all files.